docs/src/user-guide/guides-using-object-storage/index.md

```diff
@@ -27,22 +27,60 @@ will be added in a future release.
 2. An S3 bucket and [key prefix][aws-key-prefixes] containing the logs you wish to compress.
 3. An S3 bucket and key prefix where you wish to store compressed archives.
 4. An S3 bucket and key prefix where you wish to cache stream files.
-5. An AWS IAM user with the necessary permissions to access the S3 bucket(s) and prefixes mentioned
-   above.
-    * To create a user, follow [this guide][aws-create-iam-user].
-      * You don't need to assign any groups or policies to the user at this stage since we will
-        attach policies in later steps, depending on which object storage use cases you require.
-    * You may use a single IAM user for all use cases, or a separate one for each.
-    * For brevity, we'll refer to this user as the "CLP IAM user" in the rest of this guide.
-6. IAM user (long-term) credentials for the IAM user(s) created in step (4) above.
-    * To create these credentials, follow [this guide][aws-create-access-keys].
-      * Choose the "Other" use case to generate long-term credentials.
-
-    :::{note}
-    CLP currently requires IAM user (long-term) credentials to access the relevant S3 buckets.
-    Support for other authentication methods (e.g., temporary credentials) will be added in a future
-    release.
-    :::
+5. A [supported AWS authentication method](#supported-aws-authentication-methods) configured with
+   the necessary permissions to access the S3 buckets and prefixes mentioned above.
+
+   :::{note}
+   You may use a single authentication method for all the use cases above, or a separate one for
+   each.
+   :::
+
+### Supported AWS authentication methods
+
+clp-json currently supports the AWS authentication methods described below.
+
+:::{caution}
+Short-term [STS credentials][aws-sts-credentials] (which include a Session Token) are not supported
+directly. To use temporary credentials, use [named profiles](#named-profiles) (with
+IAM Identity Center authentication or IAM role assumption) to generate them dynamically instead.
+:::
```

Comment:
this part is inaccurate, because while using named profiles, the process of generating temporary credentials is hidden from the user. How about changing it to something like

```suggestion
:::{caution}
Short-term [STS credentials][aws-sts-credentials] (which include a Session Token) are not supported
directly. Instead, use [named profiles](#named-profiles) (with
IAM Identity Center authentication or IAM role assumption) which provide the required permissions without manual credential handling.
:::
```